patch_creation_wizard

```diff
@@ -302,11 +311,23 @@ def determine_manager_paths(diff_files, location_mappings):
         )
         sys.exit(1)
 
+    scp_command = [
+        'scp', '-P', ssh_details['ssh_port'],
+    ]
     base_command = [
         'ssh', '-p', ssh_details['ssh_port'], ssh_details['connection_string']
     ]
     if ssh_details['key_path']:
         base_command.extend(['-i', ssh_details['key_path']])
+        scp_command.extend(['-i', ssh_details['key_path']])
+
+    # Upload shasum script and ensure it is executable
+    scp_command.extend(
+        ['pysha256sum', ssh_details['connection_string'] + ':']
+    )
+    subprocess.check_call(scp_command)
+    chmod_command = base_command + ['chmod +x ./pysha256sum']
```

Comment:
we can `scp -p` to preserve file modes, and then we won't need to chmod+x

Reply:
Note: The word of the day is 'Tweak'